src/modules/localization/src/wheel_odom/odometry.cpp

```diff
@@ -117,10 +124,8 @@ namespace wheel_odometry_node
         }
     }
 
-    
     void WheelOdometryNode::odometry_state_estimation()

```

Comment:
Can you please mention the input and output format (dimension and/or units) for quick reference?

Reply:
Publisher/subscriber topics are included in the header file. I have included the units for the equations in the cpp file.

Reply:
Header file looks good.

On what line are the units noted in the cpp file? You may have committed and forgotten to push, or pushed to the wrong branch.